ktor-shared/ktor-serialization/common/src/SerializableWebSocketSession.kt

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright 2014-2022 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.serialization
+
+import io.ktor.websocket.*
+
+public interface SerializableWebSocketSession : WebSocketSession {
```

Comment:
the name is misleading. the session itself is not serializable, but has a serializer

Reply:
Yeah, that's true. I didn't found a better name through, maybe `WebSocketSessionWithSerializer`?